src/net_processing.cpp

```diff
@@ -35,6 +35,9 @@ extern CTweak<uint64_t> grapheneMinVersionSupported;
 extern CTweak<uint64_t> grapheneMaxVersionSupported;
 extern CTweak<uint64_t> grapheneFastFilterCompatibility;
 
+/** maximum size (in bytes) of a batched set of transactions */
+static const uint32_t MAX_TXN_BATCH_SIZE = 10000;
+
```

Comment:
would it make sense to move this declaration in the `net.h` header?

Reply:
it's only ever used in net_processing.